src/i18n/ja/nav.ts

```diff
@@ -16,7 +16,8 @@ export default NavDictionary({
 	// Tutorials
 	tutorials: 'チュートリアル',
 	'blog-tutorial': 'ブログを作る',
-	// 'island-tutorial': 'Thinking with Islands',
+	'add-collections-tutorial': 'コレクションを拡張する',
+  'add-transitions-tutorial': 'ビュートランジションを拡張する',
```

Comment:
もとのタイトルはそれぞれ

* Extend with Content Collections
* Extend with View Transitions

で、ここの with は「を」ではなく「により」みたいな感じだと思いますので、その点を反映したほうが良さそうです。また、Content Collections は日本語訳では「コンテンツコレクション」と訳しているため、その点も直したほうが良さそうです。

以上を踏まえて以下のようにするのはどうでしょうか:

```suggestion
	'add-collections-tutorial': 'コンテンツコレクションで拡張する',
  'add-transitions-tutorial': 'ビュートランジションで拡張する',
```

Reply:
そうですね、ありがとうございます！
b2d9398 で対応しました！